scopesim/tests/tests_effects/test_ReferencePixelBorder.py

```diff
@@ -51,4 +53,6 @@ def test_sets_border_to_zero(self):
             plt.imshow(implane.data, origin="bottom")
             plt.show()
 
-        assert np.sum(implane.data) == 7371
+        # Note: this used to be 7371, I don't know where that number came
+        #       from, but the current number makes more sense anyway...
+        assert np.sum(implane.data) == 7200
```

Comment:
The 10201 is 101**2, so apparently the square is 100 pixels wide and for some reason [this bug you've fixed] one pixel was added. In this test the padding is (5,5,5,15), so the 7371 comes from (100 - 5 - 5 + 1) * (100 - 5 -15 +1). Your number is correct: (100 - 5 - 5) * (100 - 5 - 15) = 7200.

Reply:
Thanks for the explanation!